### PR TITLE
Display upcoming events 408

### PR DESCRIPTION
**Pull request by Boorclark**
Fixes Issue #408

Changes Made:
-  Changed Event query 
-  Updated HTML table headers
-  Got rid of unneeded breaks

image
image

Files Changed:
 - app/controllers/main/routes.py
 - app/logic/events.py
 - app/templates/main/volunteerProfile.html

### CODE REVIEW

File: app/logic/events.py [hunk 1]
```diff
@@ -189,9 +189,9 @@ def getUpcomingEventsForUser(user,asOf=datetime.datetime.now()):
     events = (Event.select(Event)
                             .join(ProgramEvent)
                             .join(Interest, on=(ProgramEvent.program == Interest.program))
-                            .where(Interest.user == user)
-                            .where(Event.startDate >= asOf)
-                            .where(Event.timeStart > asOf.time())
+                            .where(Interest.user == user,
+                            Event.startDate >= asOf,
+                            Event.timeStart > asOf.time())
```
Comment (BrianRamsay) [inline app/logic/events.py]: Let's line these up with `Interest`, not `.where`

**Review by BrianRamsay — CHANGES_REQUESTED** [review]
Let's also reduce the header size of the message when there are no events

File: app/logic/events.py (hunk 1 above)
Comment (Boorclark) [reply app/logic/events.py]: resolved

File: app/templates/main/volunteerProfile.html [hunk 2]
```diff
@@ -53,14 +53,11 @@ <h6 align="center">{{volunteer.phoneNumber}}</h6>
            </tr>
        {% endfor %}
      {% else %}
-       <h2><p class="text-dark" align="center">Choose interests below to be notified about upcoming events.</p><h2>
+       <h4><p class="text-dark" align="center"> No upcoming events. </p></h4>
+       <h4><p class="text-dark" align="center">Choose interests below to be notified about upcoming events.</p><h4>
```
Comment (BrianRamsay) [inline app/templates/main/volunteerProfile.html]: Is there a reason we are using a paragraph tag inside a header tag? It seems a little backwards. You should be able to get the styles you want with just a header or just a paragraph

Comment (BrianRamsay) [reply app/templates/main/volunteerProfile.html]: The size seems good right now. I'm not sold on the center alignment, though.

**Review by BrianRamsay — CHANGES_REQUESTED** [review]
I added an upcoming Empty Bowls event for myself and checked notify for all the programs, but I don't see any upcoming events table

**Review by BrianRamsay — APPROVED** [review]

**Pull request merged**